File: apps/lantern-web/fireweave/fw-tracker.ts
/**
 * fw-tracker.ts — active rollout change stamps.
 *
 * `/fireweave:initialise` scaffolds this empty. Per the dev loop, each feature
 * change appends its `stmp_<ULID>` id here (the same id written to the manifest
 * `change.stampId`) so `reconcile` and the dev-checklist gates can see the stamp
 * in the committed tree.
 *
 * This is a DISK artifact, not a runtime import: nothing in the harness reads
 * it. `reconcile` and `assert_dev_checklist` read it out of the repo.
 *
 * It sits beside `fw-harness.ts` and `fw-providers.ts`, one module per surface —
 * the same shape every other language uses. Repos initialised before that
 * unification keep a `fw-tracker/index.ts` DIRECTORY elsewhere in the tree;
 * both are still read, and `/fireweave:migrate-harness` moves the old one here.
 */

/** The committed stamp record. Append one entry per feature change. */
export const FW_STAMPS: Array<{ stampId: string }> = [
  { stampId: "stmp_00MTGPIB6B9KY24ZI4" },
  { stampId: "stmp_01M1B6VMQNP3H6SZSF25" },
  { stampId: "stmp_01M1B9BG9CMXJQTD96QB" },
];

File: apps/lantern-api/src/fireweave/fw_tracker.py
"""fw_tracker.py — active rollout change stamps for the PYTHON surface.

``/fireweave:initialise`` scaffolds this empty. Per the dev loop, each feature
change appends its ``stmp_<ULID>`` id here (the same id written to the manifest
``change.stampId``) so ``reconcile`` and the dev-checklist gates can see the
stamp in the committed tree.
"""

from __future__ import annotations

FW_STAMPS: list[str] = ["stmp_01K2METRICTYPES0000000002"]
